.github/workflows/build_maru.yaml

```diff
@@ -0,0 +1,52 @@
+# Based on : https://github.com/temporalio/maru/blob/master/.github/workflows/docker.yaml
+name: Build and publish maru
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - temporal-bench
+
+jobs:
+  build:
+    name: Build and publish maru
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          repository: temporalio/maru
+    
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v5
+        with:
+          push: true  
+          tags: ghcr.io/HENNGE/maru:latest
```

Comment:
You don't need to create a repository to push this?

Reply:
Apparently not, just like dynalite which is different than it's repo name dynalite-docker
https://github.com/HENNGE/dynalite-docker/pkgs/container/dynalite